Tools/CISupport/ews-build/steps.py

```diff
@@ -5254,3 +5254,16 @@ def doStepIf(self, step):
 
     def hideStepIf(self, results, step):
         return not self.doStepIf(step)
+
+
+class DeleteStaleBuildFiles(shell.ShellCommand):
+    name = 'delete-stale-build-files'
+    description = ['Deleting stale build files']
+    descriptionDone = ['Deleted stale build files']
+    command = ['python3', 'Tools/CISupport/delete-stale-build-files', WithProperties('--platform=%(fullPlatform)s')]
```

Comment:
`delete-stale-build-files` looks like it's expecting `--platform` to be `win`. Does `fullPlatform` report `win` on Windows or a "fuller" triple like `win32-x64`?

Reply:
Yes, fullPlatform is "win". It can be verified in any ews build like https://ews-build.webkit.org/#/builders/10/builds/158586 (in properties tab)